src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -165,5 +166,14 @@
             <argument type="service" id="http_kernel" />
             <argument>%twig.exception_listener.controller%</argument>
         </service>
+
+        <service id="twig.configurator.environment" class="Symfony\Bundle\TwigBundle\DependencyInjection\Configurator\EnvironmentConfigurator">
```

Comment:
Interestingly, we cannot make the service private because it then gets inlined and the XML dumper fails here: https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/DependencyInjection/Dumper/XmlDumper.php#L182

The error message then is:

```
ContextErrorException: Warning: DOMElement::setAttribute() expects parameter 2 to be string, object given in /var/www/symfony/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Dumper/XmlDumper.php line 182
```